.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish to npm
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  cache-build:
```

Comment:
we're already doing this as part of publish release, so we could alternatively just tack this on there (i actually did it that way first), but I think I prefer this.

Reply:
If we do it as part of the `publish-release` workflow, we only have to build once. Any particular reason to do it this way?

Reply:
no real reason tbh. I can easily put these two jobs and accomplish the same thing while only building once by moving them over there. I guess that was the original intention?

Reply:
actually looking at `publish-release` now and it doesn't appear that there's a `yarn build` in there. :thinking: